lib/recipe/annotation.js

```diff
@@ -59,8 +64,9 @@ exports.annot = function annot(
 // Link, Popup, Movie, Widget, Screen, PrinterMark, TrapNet, Watermark, 3D
 exports._attachNonMarkupAnnot = function _attachNonMarkupAnnot() {};
 
-exports._annot = function _annot(subtype, args = {}, pageNumber) {
-  const { x, y, width, height, text, options } = args;
+exports._annot = function _annot(subtype, args = {}, pageNumber, ref) {
```

Comment:
has 4 function arguments but later you only call it with three, we should mark it opional. ahh ts is not supported. meh and its not public api